discovery-provider/src/queries/get_health.py

```diff
@@ -375,8 +376,7 @@ def get_health(args: GetHealthArgs, use_redis_cache: bool = True) -> Tuple[Dict,
         "final_poa_block": final_poa_block,
         "network": {
             "discovery_nodes": discovery_nodes,
-            "content_nodes": content_nodes,
-            "alive_content_nodes": alive_content_nodes,
+            "healthy_content_nodes": healthy_content_nodes,
```

Comment:
@dylanjeffers  are you okay with this change? the options are:
1. expose only healthy CNs, under this `healthy_content_nodes` field or name it `content_nodes` if you already have other code that expects that field name 
2. expose all CNs, which could include unhealthy nodes, under the `content_nodes` field name
3. expose separate fields for all CNs and healthy CNs for debugging purposes (but @raymondjacobson is concerned about that being too much data)

Reply:
yeah love these changes from content-node-selection side, we can make the changes for sdk

Reply:
if you dont mind updating StorageNodeSelector to use that new data in this pr that would be swell, can also add a commit on here if that's easiest

Reply:
IMO, we should just call this "content_nodes"

If something is unhealthy, why return it?

Reply:
sounds good! I just left it as "content_nodes" (but returning only the healthy ones) and updated StorageNodeSelector to hit /health_check instead of /status